Kwenta/kwenta
# Fix: Display margin below min deposit amount

## Description
1. Show the balance even if the available margin is below the minimum deposit amount (50 sUSD), e.g.
```
Only $18.54 available margin 
Min. $50 sUSD required to trade
```
2. Simplify the existing codes

## Related issue
Continue the work on #2631

## Motivation and Context


## How Has This Been Tested?
Case 1: The user has 0 sUSD in the available margin
- See `No available margin`
- Click on `Get sUSD` button to open `Deposit/Withdraw` modal

Case 2: The user has less than 50 sUSD in the available margin
- See `Only $xx.xx available margin`
- Click on the pencil button to open `Deposit/Withdraw` modal

Case 3: The user has more than 50 sUSD in the available margin
- See `Available margin: $xxx.xx`
- Click on the pencil button to open `Deposit/Withdraw` modal

## Screenshots (if appropriate):
***Desktop***
<img width="730" alt="截屏2023-07-25 15 21 24" src="https://github.com/Kwenta/kwenta/assets/4819006/9d07c6f8-35f0-4561-9a1c-78c8639ea3a2">

***Mobile***
<img width="500" alt="截屏2023-07-25 15 20 53" src="https://github.com/Kwenta/kwenta/assets/4819006/aebdf7a5-4040-47d5-9b82-9568eac0fdcb">
